nemo/export/tensorrt_lazy_compiler.py

```diff
@@ -189,7 +189,7 @@ def set_inputs(self, feed_dict, stream):
 
         def try_set_inputs():
             for binding in self.input_names:
-                t = feed_dict.get(self.input_table[binding], None)
+                t = feed_dict.get(self.input_table[binding], None).cuda()
```

Comment:
Well we assumed inputs should be on GPU already, is it really necessary ?

Reply:
Actually the inputs in my example are already on GPU and without cuda() call, it's giving the memory error. I guess if the tensor is already on GPU, that code shouldn't do anything anyways, no?

Reply:
Well, In PyTorch, the `cuda()` method moves a tensor to the GPU associated with the currently active CUDA device. That's apparently why you need it - are you using multiple GPUs ? 
We may do better by setting that device current - can you check with your example ?

Reply:
Actually you are right, the model wasn't on the GPU memory and the inputs were moved based on where the model is on. So, this update is not needed. Removing this update from the PR. We only need to add the onnx runtime requirement.